src/playdate/graphics.nim

```diff
@@ -166,7 +193,10 @@ proc getData*(this: LCDBitmap): BitmapData =
 
 proc getSize*(this: LCDBitmap): tuple[width: int, height: int] =
     privateAccess(PlaydateGraphics)
-    assert(this != nil)
+
+    if this.isNil:
+        assert(not this.isNil)
```

Comment:
Are you forcing a crash here to point debuggers to the cause?

Reply:
Yup -- that's the goal. But wrapping this in an `if` makes no sense. I'll fix this